app/jobs/resolution_proofing_job.rb

```diff
@@ -13,7 +13,7 @@ class ResolutionProofingJob < ApplicationJob
   )
 
   def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:)
+              dob_year_only:, user_id:, threatmetrix_session_id:)
```

Comment:
we need to be careful about adding required keyword params to an async job like this. This is the +1/-1 deploy problem, old hosts will enqueue jobs that don't have these arguments and the jobs will immediately fail

So we need to add default values of `nil` and then make sure the body of the method can handle them being nilk

```suggestion
              dob_year_only:, user_id: nil, threatmetrix_session_id: nil)
```

Reply:
yeah thought about that.... good catch